src/main/java/DialogBox.java

```diff
@@ -0,0 +1,77 @@
+import java.io.IOException;
+import java.util.Collections;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.geometry.NodeOrientation;
+import javafx.geometry.Pos;
+import javafx.scene.Node;
+import javafx.scene.control.Label;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.HBox;
+import javafx.scene.shape.Circle;
+import javafx.scene.text.Text;
+
+/**
+ * An example of a custom control using FXML.
+ * This control represents a dialog box consisting of an ImageView to represent the speaker's face and a label
+ * containing text from the speaker.
+ */
+public class DialogBox extends HBox {
+    @FXML
+    private Text dialog;
+
+    @FXML
+    private Label displayName;
+
+    @FXML
+    private ImageView displayPicture;
+
+    private Circle clip = new Circle(30, 30, 30);
+
+    private DialogBox(String text, Image img, String name) {
+        try {
+            FXMLLoader fxmlLoader = new FXMLLoader(MainWindow.class.getResource("/view/DialogBox.fxml"));
+            fxmlLoader.setController(this);
+            fxmlLoader.setRoot(this);
+            fxmlLoader.load();
+
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        displayName.setText(name);
+        dialog.setText(text);
+        displayPicture.setImage(img);
+        displayPicture.setClip(clip);
+    }
+
+    /**
+     * Flips the dialog box such that the ImageView is on the left and text on the right.
+     */
+    private void flip() {
+        String cssLayout = "-fx-background-color: whitesmoke;\n"
```

Comment:
I like how the indentation of the value for cssLayout makes it neat and easy to read.